### PR TITLE
make cargo build work

### PR DESCRIPTION
**Pull request by xcaptain**
1. Move `default-target` config to `test-kernel` package, because the `default-target` in `.cargo/config.toml` has no effect.
2. Remove `-enable-kvm` to make it runs on windows.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,3 +18,10 @@ uart_16550 = "0.2.10"
 version = "0.1.2"
 default-features = false
 features = ["regular", "size_14"]
+
+
+[profile.dev]
+panic = "abort"
+
+[profile.release]
+panic = "abort"
```
Comment (Freax13) [inline Cargo.toml]: Why are those needed?

Comment (xcaptain) [reply Cargo.toml]: At first I run `cargo build` will get some error like language item required, but not found: `eh_personality`, adding these lines fixed this error. But now I can't reproduct this error, seems not necessary, will remove later.

**Comment by Freax13** [comment]
Thank you for your contribution!
